RRemoteConfig/KeyStore.swift

```diff
@@ -53,10 +55,13 @@ internal class KeyStore {
         }
 
         if status != errSecSuccess {
+            var error: String?
             if #available(iOS 11.3, *) {
-                let error = SecCopyErrorMessageString(status, nil)
-                Logger.e("addKey error \(String(describing: error))")
+                error = SecCopyErrorMessageString(status, nil) as String?
```

Comment:
I was wondering if error can be released as it is a CFString allocation. Interesting article about memory management in swift for CF references:
https://ericasadun.com/2015/05/28/swift-five-things-about-core-foundation/

Reply:
I'll check for this in SDKCF-1442